include/zephyr/drivers/sensor.h

```diff
@@ -313,6 +320,27 @@ enum sensor_attribute {
 	/** Alert threshold or alert enable/disable */
 	SENSOR_ATTR_ALERT,
 
+	/**
+	 * To enroll fingerprint image as template
+	 */
+	SENSOR_ATTR_ADD,
```

Comment:
All these new attributes are not specific to fingerprint, any chipset support internal record/database maintenance. So please change comments accordingly and change naming as SENSOR_ATTR_RECORD_*

Reply:
okay, updated and fixed in the latest changes